packages/mask/src/plugins/Tips/SNSAdaptor/components/WalletsByNetwork.tsx

```diff
@@ -39,10 +40,14 @@ const useStyles = makeStyles()((theme) => ({
         lineHeight: '100px',
         textAlign: 'center',
     },
+    networkIcon: {
+        width: 'auto',
+        height: 18,
+    },
 }))
 
 interface WalletsByNetworkProps {
-    network: { name: string; icon: URL }
+    network: SupportedNetworkMap
```

Comment:
Pass `sufficient and necessary` data only, `networkId`, and resolve network by it.

Reply:
done